### ISSUE TITLE
libopenvr_api.so not found when running AppImage

### ISSUE THREAD

**Issue by JulianGro** [daleglass/vircadia-builder#20]
Running VR using AppImages does not work because Vircadia cannot find the `libopenvr_api.so` library.
```
[07/29 18:06:28] [INFO] [default] Loading runtime plugins from  "/tmp/.mount_VircadJP8Xoo/vircadia/interface/plugins/"
[07/29 18:06:28] [DEBUG] [hifi.plugins] Attempting plugin libopenvr.so
[07/29 18:06:28] [DEBUG] [hifi.plugins] Plugin libopenvr.so failed to load:
[07/29 18:06:28] [DEBUG] [hifi.plugins]   Cannot load library /tmp/.mount_VircadJP8Xoo/vircadia/interface/plugins/libopenvr.so: (libopenvr_api.so: Kann die Shared-Object-Datei nicht öffnen: Datei oder Verzeichnis nicht gefunden)
```

Looking at the libopenvr.so using `ldd` reveals what is going wrong.
The libopenvr.so that is in the AppImage: 
```
	libopenvr_api.so => not found
```
The libopenvr.so that is in the "normal" finished build:
```
	libopenvr_api.so => /home/motofckr9k/Vircadia/install_master/interface/plugins/./../../../vcpkg/bd824a32/installed/x64-linux/debug/lib/libopenvr_api.so (0x00007faf3ccb5000)
```
While qt5-install and vircadia get copied into the AppImage, the libopenvr_api.so is in the vcpkg folder which does not get copied over.
For reference, the library is also available in Steam `/home/motofckr9k/.steam/bin32/` and in SteamVR `/home/motofckr9k/.steam/steam/steamapps/common/SteamVR/bin/linux64/`. The location of the Steam folder is not the same on every system though.

**Comment by ChristophHaag** [comment]
For reference

SteamVR.strongly recommends using the steam runtime to run openvr apps. https://github.com/ValveSoftware/SteamVR-for-Linux#runtime-requirements

We package it for debian where it trickles down to ubuntu. It has to live in non free repos though. https://repology.org/project/openvr/versions

For an appimage it's probably still a good idea to package it.

**Comment by JulianGro** [comment]
Yeah I am not sure if using the steam runtime makes a lot of sense in our scenario.
In my case I cannot even run AppImages in there:
```
$ /home/motofckr9k/.steam/ubuntu12_32/steam-runtime/run.sh /home/motofckr9k/Applications/Vircadia-x86_64_v2020.2.5_1b8d7e8d7dba7f649ab87f1ec475b9a7.AppImage
/usr/bin/AppImageLauncher: /home/motofckr9k/.steam/ubuntu12_32/steam-runtime/pinned_libs_64/libcurl.so.4: version `CURL_OPENSSL_4' not found (required by /usr/bin/../lib/x86_64-linux-gnu/appimagelauncher/libappimageupdate.so)
$
```

**Comment by daleglass** [comment]
Attempted to address this in commit 26dc453

The issue here is that plugins aren't linked to the main binary and so not found during a dependency search, that commit should rectify that.

**Comment by JulianGro** [comment]
Fixed by 26dc453313cef9cf756b7a7cc73e5aba56de00d0 :+1:
